graphql-lwt/src/graphql_lwt.ml

```diff
@@ -34,7 +34,7 @@ module Server = struct
         C.Server.respond_string ~status:`OK ~body ()
     | Error err ->
         let body = Yojson.Basic.to_string err in
-        C.Server.respond_error ~body ()
+        C.Server.respond_string ~status:`Internal_server_error ~body ()
```

Comment:
should these also add a `Content-Type` header?

Reply:
That would be nice, but I'm fine leaving that for a separate PR.